### ISSUE TITLE
remove dotted outline from icons

### ISSUE THREAD

**Issue by janpaepke** [grayghostvisuals/transformicons#36]
I think we should remove the dotted outline from the .tcon class.
see here: https://github.com/grayghostvisuals/transformicons/blob/master/dist/transformicons/base/_global-styles.scss#L19
example:
![image](https://cloud.githubusercontent.com/assets/3076177/6124019/e5fbea74-b109-11e4-9392-8d0bcc9736da.png)

I can't imagine a scenario were this would be visually desired.

The intention may be to provide visual feedback for keyboard-only users but at what cost?
If we decide we definitely want to provide focus styles they visually should fit the icon.
For example changing the color or overall scale.

Long story short:
I think we should either remove it or make it more fitting.

**Comment by grayghostvisuals** [comment]
I think make it more fitting is the for the best.

**Comment by jfroffice** [comment]
:+1: to remove the outline.

Maybe we could make something like a 'material ripple effect' when an user click on a button.
http://codepen.io/Craigtut/pen/dIfzv

**Comment by merodriguezblanco** [comment]
@grayghostvisuals please see #68  ;)
